commune_app/tests/test_user_functions.py

```diff
@@ -61,18 +65,44 @@ def chore0(user0):
     return chore0
 
 
+@pytest.fixture
+@pytest.mark.django_db()
+def chore1(user0):
+    chore1 = Chore(
+        title=TITLE,
+        description=DESCRIPTION,
+        date=DATE,
+        budget=BUDGET,
+        assign_to=user0,
+        passed=NOT_PASSED,
+        completed=COMPLETED
+        )
+    chore1.save()
+    return chore1
+
+
 @pytest.mark.django_db()
 class TestUserFunctions:
 
     def test_join_commune(self, user0, commune0):
-        user0.join_commune(commune0.id)
+        assert user0 not in User.objects.filter(commune_id=commune0)
+        join_string = user0.join_commune(commune0.id)
         assert user0 in User.objects.filter(commune_id=commune0)
+        assert join_string == "you joined to commune: " + commune0.name
 
     def test_leave_commune(self, user0, commune0):
         user0.join_commune(commune0.id)
+        assert user0 in User.objects.filter(commune_id=commune0)
         user0.leave_commune()
-        assert (user0 in User.objects.filter(commune_id=commune0)) is False
+        assert user0 not in User.objects.filter(commune_id=commune0)
 
     def test_execute_chore(self, chore0, user0):
+        assert not chore0.completed
         chore0.execute_chore(user0.id)
         assert chore0.completed
+
+    def test_user_vote(self, chore1, user0):
+        assert not chore1.passed
+        vote = Vote.create_new_vote(user0, chore1, True)
```

Comment:
it will be more readable like:
```suggestion
        vote = Vote.create_new_vote(voting_user=user0, voted_chore=chore1, vote_bool=True)
```